### PR TITLE
fix: consume messages on a given topic

### PR DESCRIPTION
**Pull request by dhmlau**
Following the instruction in the README, it complains about the GET endpoint for `/topics/{topic}/messages` doesn't exist.  This PR adds the implementation of this endpoint. 

@raymondfeng, one thing I'm not sure is that whether I should be creating a new client everytime.  But then the existing function to create client requires a groupID.  Thanks.

### CODE REVIEW

File: src/controllers/kafka-demo.controller.ts [hunk 1]
```diff
@@ -239,4 +240,44 @@ export class KafkaDemoController {
...
+    consumer.on('message', message => {
+      count++;
+      response.write(`id: ${message.offset}\n`);
+      response.write('event: message\n');
+      response.write(`data: ${JSON.stringify(message)}\n`);
+      if (count >= limit) {
+        response.end();
+      }
+    });
+    return response;
```
Comment (raymondfeng) [inline src/controllers/kafka-demo.controller.ts]: The logic to fetch a number of messages for a consumer is duplicate. Can we refactor it into a private method?

File: src/controllers/kafka-demo.controller.ts [hunk 2]
```diff
@@ -239,4 +240,44 @@ export class KafkaDemoController {
...
+      },
+    },
+  })
+  async consumeMessagesOnTopics(
+    @param.path.string('topic') topic: string,
+    @param.query.string('limit') limit: number,
+    @inject(RestBindings.Http.RESPONSE) response: Response,
+  ) {
+    let client = new KafkaClient();
+    let consumer = new Consumer(client, [{topic: topic}], {autoCommit: false});
```
Comment (raymondfeng) [inline src/controllers/kafka-demo.controller.ts]: Maybe you can simply use `createConsumer`.

File: src/controllers/kafka-demo.controller.ts (hunk 1 above)
Comment (raymondfeng) [inline src/controllers/kafka-demo.controller.ts]: I guess we should close the consumer after each call.

Comment (dhmlau) [reply src/controllers/kafka-demo.controller.ts]: added `writeMessageToResponse` function.

**Review by raymondfeng — APPROVED** [review]

**Pull request merged**